README.md

```diff
@@ -1,8 +1,6 @@
-**This extension is now obsolete as the gnome calculator provides this functionality since g-s 3.14**
```

Comment:
I know that GNOME Calculator provides this feature out-of-the-box, but unfortunately this is not working in all distros (e.g. Ubuntu 18.04).